content/blog/2016-07-23-pull-does-not-scale-or-does-it.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Pull doesn't scale - or does it?
+created_at: 2016-07-23
+kind: article
+author_name: Julius Volz
+---
+
+Let's talk about one particularly hard-to-kill myth again. Whenever there is a
```

Comment:
I suggest to use a word like "misconception" instead of "myth". As somebody who did serious academic work on myths, I always cringe if it is used as a synonym of "misconception". It's really like using "hacker" for "criminal that breaks into computer system for personal gain".